Source/d_main.c

```diff
@@ -133,16 +133,16 @@ char    basesavegame[PATH_MAX+1];  // killough 2/16/98: savegame directory
 //jff 4/19/98 list of standard IWAD names
 const char *const standard_iwads[]=
 {
-  "/doom2f.wad",
-  "/doom2.wad",
-  "/plutonia.wad",
-  "/tnt.wad",
-  "/doom.wad",
-  "/doom1.wad",
+  "doom2f.wad",
```

Comment:
Hm, another thought. Since the `DIR_SEPARATOR_S` macro is a string literal, couldn't we just slam it in front if each entry in `standard_iwads[]`?

Reply:
For example 
```
DIR_SEPARATOR_S "doom2f.wad",
```

Reply:
This would change more net lines of code but would let the logic untouched.

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/172/commits/04710a6705be3cf9cb764690c90382e8ef17d771